### DISCUSSION TITLE
Advice on Running RCT1 and RCT2 on Windows 10?

### DISCUSSION THREAD

**Discussion by Derpercoaster** [OpenRCT2/OpenRCT2#18740] [Q&A]
Running Rollercoaster Tycoon 1 and 2 on Windows 10 is a nightmare for me. I can't get the games to load at all sometimes, and when they boot, there's a huge possibility of crashes, and even hardlocks (which require me to hard-reset my PC every time) from seemingly harmless things (I got the game to hardlock by changing the screen resolution. Seriously). Windows compatibility does not make things any easier, and third party programs are few and far between, if they are even safe to use. Is there an easy way, or even ANY way, to run these games on CD or Steam without all of these ridiculous issues?

**Comment by ocalhoun6** [comment]
You might try...

- Running it on a Windows XP virtual machine, including setting up the virtual machine to have the kind of native resolution RCT 1 and 2 expect. Since the game has very low system requirements, VM performance shouldn't be an issue.
- Running it on Linux through wine. Wine is often better at supporting legacy Windows programs than Windows itself is these days, and I hear that RCT2 runs quite well through wine.

**Reply by Derpercoaster** [reply]
I have tried using a Windows XP VM to run it, but it has most of the same problems that it does in WIndows 10, at least in the VM I used. The game crashed often for no apparent reason, and even hardlocked a couple of times (requiring a the VM to be reset). Linux might work, but I have no experience with it.
